src/oom/oomd-util.c

```diff
@@ -355,10 +357,10 @@ int oomd_cgroup_context_acquire(const char *path, OomdCGroupContext **ret) {
         if (r < 0)
                 return log_debug_errno(r, "Error parsing memory pressure from %s: %m", p);
 
-        r = cg_get_owner(SYSTEMD_CGROUP_CONTROLLER, path, &uid);
-        if (r < 0)
-                log_debug_errno(r, "Failed to get owner/group from %s: %m", path);
-        else if (uid == 0) {
+        if ((r = cg_get_owner(SYSTEMD_CGROUP_CONTROLLER, path, &cg_uid)) < 0 ||
+            (r = cg_get_owner(SYSTEMD_CGROUP_CONTROLLER, monitored_ancestor_path, &ancestor_cg_uid)) < 0)
+                log_debug_errno(r, "Failed to get owner/group from %s and %s: %m", path, monitored_ancestor_path);
```

Comment:
and -> and/or (?)
Alternatively split these conditions up and log the messages separately